### DISCUSSION TITLE
Notify when a change is made in a library

### DISCUSSION THREAD

**Discussion by tidyzilla** [storybookjs/storybook#26573] [Help] [answered]
### Summary

Hi,

Sorry, maybe dumb question here, but I couldn't find anything about it.

I am a designer, and in my company, devs use Storybook to document what components exists. Problem is when they push a new component or edit one, unless they communicate it, we're not updated of those changes.

Is there a way to receive some notifications, emails, whatever...  to be aware of changes made to a library? Would be awesome to get notified when a component has been updated or deleted, a new one published...

Thanks

**Comment by valentinpalkovic** [comment, marked as answer]
Hi @tidyzilla 

Thank you for your excellent question!

For staying updated on changes to your component library, such as new updates or the addition of new components, integrating a visual regression testing tool can serve as an effective solution. Visual regression testing helps in identifying and documenting any changes in the appearance of components by comparing the visual aspects of components before and after updates.

One tool that facilitates this, along with offering the specific notification features you're looking for, is [Chromatic](https://www.chromatic.com). Chromatic automates visual testing for web UIs, ensuring that your components look as intended across different environments. It's built by the folks behind Storybook, which means it integrates seamlessly with your existing setup.

With Chromatic, each time a component is updated or added, the tool automatically detects these changes during its visual testing process. You can configure Chromatic to notify team members via email or through integrations with Slack, GitHub, and other tools whenever there are changes in the visual appearance of components. This means you and your design team can stay informed about any updates without needing developers to notify you manually and even stop the developers of merging PRs if the visual state of a component isn't as intended or gets corrupted.

By relying on Chromatic for visual regression testing and its notification capabilities, your workflow can become more streamlined, ensuring the design team is always aligned with the developments happening in the component library. This approach not only saves time but also enhances collaboration between designers and developers.

**Reply by tidyzilla** [reply]
Awesome, thanks !
